docs/package.json

```diff
@@ -9,7 +9,7 @@
     "build-sw": "node ./scripts/buildServiceWorker.js",
     "dev": "rimraf ./node_modules/.cache/babel-loader && cross-env NODE_ENV=development node src/server.js",
     "deploy": "git push material-ui-docs master:latest",
-    "export": "rimraf docs/export && next export --threads=3 --concurrency=5 -o export && yarn build-sw && cpy cdn/. export",
+    "export": "rimraf docs/export && next export --threads=3 -o export && yarn build-sw",
```

Comment:
The concurrency argument was added to speed up the build with Next.js v8. As far as I remember, it was speeding the build up. But now, if we look at the build duration:

> Finished processing build request in 12m4.329294595s

It seems to be about the same between Next.js v8 and v9. It doesn't seem required anymore :+1:.

Reply:
It's not supported anymore anyway.